fastchat/conversation.py

```diff
@@ -1115,6 +1112,7 @@ def get_conv_template(name: str) -> Conversation:
         roles=("Human", "Assistant"),
         sep_style=SeparatorStyle.ADD_COLON_SINGLE,
         sep="\n\n",
+        max_image_size_mb=5 / 1.35,
```

Comment:
The maximum image size for claude is given by the base64 encoded image. The base64 encoded image is ~1.33x the size of the original image. Therefore, we have to do this sorta hacky approach of doing 5 MB / 1.35 as a ratio.